pypdf/generic/_data_structures.py

```diff
@@ -263,6 +264,40 @@ def read_from_stream(
 
 
 class DictionaryObject(dict[Any, Any], PdfObject):
+
+    _init_alt_arg_names_: Optional[dict[str, str]] = None
+
+
+    """
```

Comment:
The documentation seems to be somehow located at the wrong place. It should go into the methods, not before them.